tests/management/group/test_definer.py

```diff
@@ -92,3 +92,20 @@ def modify_default_group(self, system=True):
 
             group.system = system
             group.save()
+
+    def test_admin_default_group_seeding_properly(self):
+        """Test that admin default group are seeded properly."""
+        with tenant_context(self.public_tenant):
```

Comment:
With the changes in #660, do we need `tenant_context` here for this test to pass since we're just using the public schema anyway? If not, we should probably just leave it out if possible.

Reply:
I removed the `tenant_context` from the test and it still passed.